src/styles/main.scss

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
 }
+
+$grey: #eee;
+$square-side: 100px;
+$calendar-gap: 1px;
+$calendar-day-border: 1px;
+$calendar-day-space: $square-side + $calendar-gap;
+$calendar-padding: 10px;
+$calendar-width: ($square-side + $calendar-gap) * 7;
+
+.calendar {
+  width: $calendar-width;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+  padding-top: $calendar-padding * 3;
```

Comment:
I don't think this two variables make sense

Reply:
> I don't think this two variables make sense

I was apply one padding value